src/components/ManageNoteForm.js

```diff
@@ -7,76 +7,32 @@ import Stack from '@mui/material/Stack';
 import AddCircleIcon from '@mui/icons-material/AddCircle';
 import CancelIcon from '@mui/icons-material/Cancel';
 import Grid from '@mui/material/Grid';
-import { styled } from '@mui/material/styles';
-import Dialog from '@mui/material/Dialog';
-import PropTypes from 'prop-types';
-import IconButton from '@mui/material/IconButton';
-import CloseIcon from '@mui/icons-material/Close';
-import DialogTitle from '@mui/material/DialogTitle';
 import DialogContent from '@mui/material/DialogContent';
 
-const BootstrapDialog = styled(Dialog)(({ theme }) => ({
-  '& .MuiDialogContent-root': {
-    padding: theme.spacing(2)
-  },
-  '& .MuiDialogActions-root': {
-    padding: theme.spacing(1),
-  },
-}));
-const BootstrapDialogTitle = (props) => {
-  const { children, onClose, ...other } = props;
-  return (
-    <DialogTitle sx={{ m: 0, p: 2 }} {...other}>
-      {children}
-      {onClose ? (
-        <IconButton
-          aria-label="close"
-          onClick={onClose}
-          sx={{
-            position: 'absolute',
-            right: 8,
-            top: 8,
-            color: (theme) => theme.palette.grey[500],
-          }}
-        >
-          <CloseIcon />
-        </IconButton>
-      ) : null}
-    </DialogTitle>
-  );
-};
-BootstrapDialogTitle.propTypes = {
-  children: PropTypes.node,
-  onClose: PropTypes.func.isRequired,
-};  
-const AddForm = ({onAdd,open,setOpen}) => {
-  const [title, setTitle] = useState('');
-  const [content, setContent] = useState('');
+const ManageNoteForm = ({onAdd,onEdit,newForm,onCloseModal}) => {
+  const [title, setTitle] = useState(newForm.title)
+  const [content, setContent] = useState(newForm.content)
   const onSubmit = (e) => {
-    e.preventDefault()
-    onAdd({title,content})
-    setOpen(false)
-    setTitle('')
-    setContent('')
+    e.preventDefault();
+    setTitle('');
+    setContent('');
+    if(newForm.id){
+      onEdit(title, content)
+    }
+    else{
+      onAdd({title,content})
+    }
```

Comment:
check the functions onEdit and onAdd, the first one passes 2 arguments,  but the second one passes an object. Please make it consistent in both scenarios.